apps/api/src/app/integrations/usecases/get-decrypted-integrations/get-decrypted-integrations.usecase.ts

```diff
@@ -29,6 +29,10 @@ export class GetDecryptedIntegrations {
       : await this.integrationRepository.find(query);
 
     return integrations.map((integration: IntegrationEntity) => {
+      if (!integration) {
+        return null;
+      }
```

Comment:
```suggestion
    return integrations.filter(integration => integration).map((integration: IntegrationEntity) => {
```

Maybe worth just filtering it out, or even better throw and error before this map?